src/configure/configure.ts

```diff
@@ -64,9 +64,13 @@ export async function configurePipeline(node: AzureTreeItem) {
         }
         catch (error) {
             if (!(error instanceof UserCancelledError)) {
-                extensionVariables.outputChannel.appendLine(error.message);
                 vscode.window.showErrorMessage(error.message);
-                telemetryHelper.setResult(Result.Failed, error);
+                if (extensionVariables.isErrorWhitelisted === true) {
+                    telemetryHelper.setResult(Result.Succeeded);
+                } else {
+                    extensionVariables.outputChannel.appendLine(error.message);
```

Comment:
what is this for? any idea?

Reply:
Not sure about the role, but since found at every place where result is being set, have made it common here as well.